account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_parser.py

```diff
@@ -93,7 +95,9 @@ def _data_dict_constructor(self, mapping, header):
             "bank_account_column",
         ]
         data_dict = {}
-        for key in list_of_content:
+        for key in required_list:
+            data_dict[key] = header.index(getattr(mapping, key))
```

Comment:
What about the raise if not found?

Reply:
AFAICS, if the element is not found, the `index()` method will raise the exception, which won't be catched, so it is what we want...

Reply:
OK